docs/pages/admin-guides/management/guides/aws-iam-identity-center.mdx

```diff
@@ -187,39 +192,79 @@ as usual.
-as necessary after the initial sync completes.
+Teleport creates an Access List for each group imported from the Identity Center
+instance, with group members becoming Access List members. Default Access List 
+owners are configured during the initial integration enrollment flow and can be
+adjusted as necessary after the initial sync completes.
 
 Each imported Access List is automatically assigned a role (or a set of roles)
-that grant all members of that list access to a particular permission set on a
-particular AWS account based on the permissions the corresponding Identity Center
-group was assigned during the integration setup. Those roles are considered
-system roles generated by Teleport and are named using `<permission-set-name>-on-<account-name>`
-convention (e.g. `AdministratorAccess-on-my-account`).
+that grant all members of that list access to all of the Account Assignments
+assigned to the corresponding AWS Identity Center group during the integration
+setup. 
+
+These Teleport-generated roles each represent a single Account Assignment, and
+are named using `<permission-set-name>-on-<account-name>-<account-id>` convention
+(e.g. `AdministratorAccess-on-MyAccount-012345678`). 
+
+<Admonition type="warning">
+These roles are considered system roles, and any edits or updates to them will
+be automatically reverted. 
+</Admonition>
 
 To give a user permission granted by an already-existing Identity Center synced
 Access List, an owner can add that user as a member which makes Teleport to add
 the user to its corresponding Identity Center group.
 
-<Admonition type="note">
-While the integration is running, all existing Teleport users are synced to
-Identity Center.
+<Admonition type="warning">
+Currently all existing Teleport users are synced to Identity Center. Label-based
+user filtering will be supported in a later release.
 </Admonition>
 
 Removing a member from an Identity Center synced Access List removes them
 from the corresponding Identity Center group effectively revoking privileges.
 
 In addition to membership changes, Teleport propagates changes in Access List
-grants to Identity Center as well. In a scenario where, say, for an Access List
-with roles `AdministratorAccess-on-my-account` and `ReadOnlyAccess-on-my-account`
-one of the granted roles were to be removed, the corresponding Identity Center
-group would see its assignments updated accordingly.
+grants back to Identity Center as well. For example, imagine an Access List with
+the roles `AdminAccess-on-my-account` and `ReadOnlyAccess-on-my-account`. If the
+Access List owner removes the `AdminAccess-on-my-account` role from the Access Lists,
+that change will be propagated back to AWS and the corresponding Identity Center
+group will have its assignments updated to remove the `AdminAccess` Permission 
+
+### Just-in-time Access with Resource Access Requests
+
+Teleport represents the imported AWS accounts as Apps in the Teleport Resource
```

Comment:
```suggestion
Teleport represents the imported AWS accounts as apps in the Teleport Resource
```
We haven't been capitalizing the names of resourced types like apps and databases.